### DISCUSSION TITLE
Table is showing a mix of light and dark theme

### DISCUSSION THREAD

**Discussion by FuriousBranko** [saadeghi/daisyui#1708] [Q&A] [answered]
Hello,
I'm using NextJS 13, and I've set the data-theme to be 'light' and the daisyui config to use the 'light' theme as the darkTheme.

Since my system is set to Dark mode, it keeps using the text color from the dark theme and the rest of the colors from the light theme.
![image](https://user-images.githubusercontent.com/36929510/227537419-ef9fbb55-a653-4280-a837-832806c0151e.png)
![image](https://user-images.githubusercontent.com/36929510/227537557-d28c1444-1c8c-476d-b572-40a67fd0d068.png)

tailwind.config.js:
```
module.exports = {
    plugins: [require('daisyui'), require('prettier-plugin-tailwindcss')],
    daisyui: {
        darkTheme: 'light',
    },
};
```

NextJS _document:
```
import { Html, Head, Main, NextScript } from 'next/document';

export default function Document() {
    return (
        <Html lang="en" data-theme="light">
            <Head />
            <body>
                <Main />
                <NextScript />
            </body>
        </Html>
    );
}
```

**Comment by saadeghi** [comment, marked as answer]
Looking at your screenshot, I see that you have other sources or CSS in your page that apply styles for colors. Those variables are not from either Tailwind CSS (by default) or daisyUI so I guess that must be the problem.    
<img width="638" alt="13" src="https://user-images.githubusercontent.com/7342023/227712302-d29d3c2e-2b72-43e0-bd9a-d47f5ff266b9.png">

Also if you have any `dark:` Tailwind prefix class name, that might also override the daisyUI colors so be careful about that.

Let me know if you have any questions.
